src/main/kotlin/com/autonomousapps/internal/advice/ConsoleReport.kt

```diff
@@ -1,16 +1,15 @@
 package com.autonomousapps.internal.advice
 
-import com.autonomousapps.advice.Advice
 import com.autonomousapps.advice.Dependency
 
 internal data class ConsoleReport(
-  val addToApiAdvice: Set<Advice>,
-  val addToImplAdvice: Set<Advice>,
-  val removeAdvice: Set<Advice>,
-  val changeToApiAdvice: Set<Advice>,
-  val changeToImplAdvice: Set<Advice>,
-  val compileOnlyDependencies: Set<Dependency>,
-  val unusedProcsAdvice: Set<Advice>
+  val addToApiAdvice: List<Dependency>,
```

Comment:
is there a reason you changed from set to list?

Reply:
There was a reason, but I forgot wich one. Do you prefer sets ?